Misc/NEWS.d/next/Library/2019-05-22-15-26-08.bpo-37008.WPbv31.rst

```diff
@@ -0,0 +1,2 @@
+Add support for calling `next` with the mock resulting from
```

Comment:
Sorry, I should have been more specific. This should probably be ```:func:`next` ``` or double backtick.
I'd suggest ``` :func:`next` ```

Reply:
My bad. Fixed.